### DISCUSSION TITLE
配置nvim_tree插件时，setup 函数报错

### DISCUSSION THREAD

**Discussion by yaotutu** [nshen/learn-neovim-lua#140] [Q&A] [answered]
* nvim babe 0.6.1
* 经过排查，只要执行 xxx.setup()函数就会报错，删掉该函数就会正常
*
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/21394924/201482805-b1bc311c-a295-4931-abb6-3f801ee9ff0e.png">

**Comment by nshen** [comment, marked as answer]
nvim-tree 要升到 nvim 0.7 才行

**Reply by yaotutu** [reply]
感谢,升级到最新版 0.91,问题已经解决了.
